test/onnx/internal/fx/test_frontend_consistency.py

```diff
@@ -0,0 +1,157 @@
+# Owner(s): ["module: onnx"]
+
+"""Test consistency between the output values of fx.GraphModule produced by FX frontend
+and torch.nn.Modules.
+"""
+
+from __future__ import annotations
+
+import functools
+from typing import Sequence, Union
+
+import torch
+from torch import nn
+from torch.onnx._internal.fx import frontend
+from torch.testing._internal import common_device_type, common_modules, common_utils
+from torch.utils import _pytree as pytree
+
+_FrontendType = Union[frontend.FxFrontend, frontend.FxFrontendUnpackKwargs]
+FX_FRONTENDS: Sequence[_FrontendType] = [
+    # TODO(bowbao): skip aotautograd, because it alters graph inputs/outputs.
```

Comment:
Can you test this a simple nn.Module or a 1-op function so that aotautograd doesn't alter the function? This makes me  feel we need to have the ability to select what frontend to test for which model.

Reply:
> Can you test this a simple nn.Module or a 1-op function so that aotautograd doesn't alter the function?

I think it only works on models with no parameters.

> This makes me feel we need to have the ability to select what frontend to test for which model.

This is required. More like a decorator of `expect_failure` based on the frontend. It's part of TODO.